astro/src/content/docs/extend/events-and-webhooks/webhook-event-log.mdx

```diff
@@ -0,0 +1,70 @@
+import DiagramTxAny from 'src/diagrams/docs/extend/events-and-webhooks/_transaction-any-succeed.astro';
+import DiagramTxThree from 'src/diagrams/docs/extend/events-and-webhooks/_transaction-any-succeed-three-failures.astro';
+import { YouTube } from '@astro-community/astro-embed-youtube';
+
+<Aside type="version">
+Available since 1.53.0
+</Aside>
+
+## Overview
+
+A Webhook Event Log is provided to review events sent by FusionAuth along with timing and result information for each attempt to send the event to configured endpoints. The `webhook_event_log_viewer` role grants users access to these pages. You can find the Webhook Event Log by navigating to <Breadcrumb>System > Webhook Log</Breadcrumb>. The same information is available via the [Webhook Event Log APIs](/docs/apis/webhook-event-logs).
+
+## List of Webhook Event Logs
+
+The main Webhook Event Log page provides a list of summary information and search functionality for events.
+
+<img src="/img/docs/extend/events-and-webhooks/webhook-event-log-list.png" alt="List of Webhook Event Logs" width="1200" role="bottom-cropped" />
+
+Using the icons on this screen, you can:
+
+* <Icon name="address-card" /> Manage the webhook event log
+
+## Manage Webhook Event
+
+Click the <InlineUIElement>Manage</InlineUIElement> button on the list to manage a webhook event.
+
+The page displays summary information about the event such as timing, event type, and result. The <InlineUIElement>Webhook attempts</InlineUIElement> tab contains information on attempts to send the event to each configured receiver.
+
+<img src="/img/docs/extend/events-and-webhooks/manage-webhook-event.png" alt="Manage of Webhook Event" width="1200" role="bottom-cropped" />
+
+Using the icons on this screen, you can:
+
+* <Icon name="fa-search" /> View attempt details
+
+The <InlineUIElement>Source</InlineUIElement> tab shows a readonly view of the event request body. The same event body is sent to all receivers.
+
+<img src="/img/docs/extend/events-and-webhooks/manage-webhook-event-source.png" alt="Webhook Event Source" width="1200" />
+
+## View attempt details
+
+Click the <InlineUIElement>View</InlineUIElement> button on the manage screen to view details for an attempt to send the event.
+
+<img src="/img/docs/extend/events-and-webhooks/webhook-attempt.png" alt="Webhook Attempt Log" width="1200" role="bottom-cropped" />
+
+## Configuration
+
+By default Webhook Event Logs are never deleted. The retention period can be configured under <Breadcrumb>Settings > System</Breadcrumb> on the <InlineUIElement>Advanced</InlineUIElement> tab.
+
+<img src="/img/docs/extend/events-and-webhooks/webhook-event-log-settings.png" alt="Webhook Event Log Settings" width="1200" />
+
+When the <InlineField>Delete</InlineField> retention is enabled, Webhook Event Logs older than the configured number of days will be deleted automatically.
+
+## Limitations
```

Comment:
Suggest removing this section altogether

Reply:
Why mix it in rather than having a section?

Reply:
I don't actually think it is a "limitation", it's just the way the retention works.

Reply:
Isn't that true of all "limitations"? :smart: